### ISSUE TITLE
get_project_borehole_data() AttributeError when no ZIP-file in project_info["report"]

### ISSUE THREAD

**Issue by danpejobo** [emerald-geomodelling/libnadagclient#6]
This posted error is from a newly created virtual environment. I also tested it on my windows environment , same error.

python --version
```
Python 3.10.1
```

pip freeze -l
```
cchardet==2.1.7
libnadagclient @ git+https://github.com/emerald-geomodelling/libnadagclient.git@a4ec5c1bbc56fb12abf49fa63b89ad45663c787d
libsgfdata==0.0.12
lxml==4.7.1
numpy==1.22.0
OWSLib==0.25.0
pandas==1.3.5
python-dateutil==2.8.2
python-slugify==5.0.2
pytz==2021.3
requests-html==0.10.0
requests==2.27.1
six==1.16.0
text-unidecode==1.3
urllib3==1.26.8
```

The NADAG id is randomly chosen. **I tried again with the ID from the readme file and that worked!**
```
Python 3.10.1 (main, Dec 18 2021, 23:53:45) [GCC 11.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import libnadagclient
>>> libnadagclient.get_project_id('R-1069')
'416e31bf-80a7-4652-80a8-9777c31284d4'
>>> d = libnadagclient.get_project_borehole_data("416e31bf-80a7-4652-80a8-9777c31284d4")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/libnadagclient/__init__.py", line 108, in get_project_borehole_data
    for url in _get_project_zip_files(get_project_info(project_id)):
  File "/usr/lib/python3.10/site-packages/libnadagclient/__init__.py", line 94, in _get_project_zip_files
    return {v for k, v in project_info["report"].items() if k.lower().endswith(".zip")}
AttributeError: 'str' object has no attribute 'items'
```

**Comment by danpejobo** [comment]
The problem seems to be that when _get_project_zip_files(project_info) method looks up the key "report" and the item is empety in the project_info, there is nothing to get

```
>>> libnadagclient.get_project_info("416e31bf-80a7-4652-80a8-9777c31284d4")
{'oppdragstaker': 'Ukjent', 
'oppdragsgiver': 'Ikke oppgitt', 
'prosjektnavn': 'Geoteknisk undersøkelse fra Oslo kommune. ID_Saksnummer: R-1069', 
'prosjektnr': 'R-1069', 
'antallborehull': {'2350': 'https://geo.ngu.no/api/faktaark/nadag/visGeotekniskUndersokelseBorehull.php?id=416e31bf-80a7-4652-80a8-9777c31284d4'}, 
'beskrivelse': 'Geoteknisk undersøkelse med borehull fra Oslo kommune med ID_Saksnummer. For ytterlige opplysninger send adressen eller kartutsnittet til Oslo kommune (dig.kart@pbe.oslo.kommune.no)', 
'boreprofil': '', 
'nedlastbare': 'Prosjektet er ikke tilgjengelig for nedlasting ennå', 
'report': '', 
'datoformat': '08.05.2020'}
```
Maybe throw in a try-except block in the function? and then message to the logger in a correct way what went wrong, then return the error and handle it in the get_project_borehole_data() method?
```
def _get_project_zip_files(project_info):
    try:
        return {v for k, v in project_info["report"].items() if k.lower().endswith(".zip")}
    except Exception as e:
        logger.warn("Just an empety string/No zip file in.... %s:%s: %s" % (xxx, xxx, e))
        return e
```

Or use the dictionary get("report") function, which return none (as default) if there is no data, and then handle the none in the get_project_borehole_data() method. Also need to handle the return of empety string...

**Comment by redhog** [comment]
I have published a new version (0.0.3) that fixes this, and also pulls out as much data as it can from the borehole pages at nadag in the case where zip files of sgf format files are missing (Please see the other ticket linked from the pull request).
